src/loader/endpoints/corrections.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd

```

Comment:
The file name could be a bit more descriptive. Corrections of what? 

If is just to correct inloco data, why not to put inside the inloco .py files?

If can be generalized, why not to put the functions inside `utils.py`?